htmlhint/vscode-htmlhint
# Fixes for compatibility with latest VS Code

- update lock files to latest versions
- add GitHub's Super Linter GitHub Action
- Add Dependency Review GitHub Action
- update Dependabot config to check both directories
- Various small fixes
